dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ESContentletIndexAPITest.java

```diff
@@ -571,23 +571,18 @@ public void testSearch () throws Exception {
 
             //++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
             //++++++++++++++++++++++++OTHER TESTS+++++++++++++++++++++++
-            //Testing the search
-            siteSearchResults = siteSearchAPI.search( indexName, "engli*", 0, 100 );
+            //Testing the search by existing word with wildcard
+            siteSearchResults = siteSearchAPI.search( indexName, "illustrati*", 0, 100 );
             //Validations
             assertTrue( siteSearchResults.getError() == null || siteSearchResults.getError().isEmpty() );
             assertTrue( siteSearchResults.getTotalResults() > 0 );
 
-            //Testing the search
-            siteSearchResults = siteSearchAPI.search( indexName, "*engli", 0, 100 );
+            //Testing the search: existing incomplete word with no wildcard
+            siteSearchResults = siteSearchAPI.search( indexName, "llustrat", 0, 100 );
```

Comment:
and let's change this one to expect 1 instead of 0. Also please change de order of the parameters. The Expected value should be the first parameter. @acurionedotcms